impress-org/give-double-the-donation
# Feature: Email confirmation

## Description

This PR adds Email tags `{matching_company}` and `{matching_company_guidelines}`

## Affects

Email template


## Testing Instructions

Add `{matching_company}` and `{matching_company_guidelines}` to the email template.
Make a donation
Check email

## Pre-review Checklist
